### PR TITLE
dev/reference: update DDL information

### PR DESCRIPTION
**Pull request by zimulala**
-  Add the latest description for `change/modify column`

- Update details for `create table`

### CODE REVIEW

File: dev/reference/sql/statements/ddl.md [hunk 1]
```diff
@@ -170,14 +172,21 @@ table_option:
...
     |`AUTO_INCREMENT`|自增字段初始值                        |`AUTO_INCREMENT` = 5|
+    |`SHARD_ROW_ID_BITS`|用来设置隐式 _tidb_rowid 的分片数量的 bit 位数 |`SHARD_ROW_ID_BITS` = 4|
+    |`PRE_SPLIT_REGIONS`|用来设置预先建立 Region 的个数 |`PRE_SPLIT_REGIONS` = 4|
     |`CHARACTER SET` |指定该表的字符串编码。目前支持 UTF8MB4| `CHARACTER SET` =  'utf8mb4'|
     |`COMMENT`       |注释信息                              | `COMMENT` = 'comment info' |
 
+
+* `split-table` 配置项默认情况下会开启，在此配置项开启时，建表操作会为每个表建立单独的 Region。
+
+
```
Comment (bb7133) [inline dev/reference/sql/statements/ddl.md]: 一个空行应该就可以了

File: dev/reference/sql/statements/ddl.md [hunk 2]
```diff
@@ -170,14 +172,21 @@ table_option:
...
 
-* `table_option` 目前支持的只有 `AUTO_INCREMENT`，`CHARACTER SET` 和 `COMMENT`，其它只是语法上支持。具体内容参考下表，各个子句之间用逗号隔开。
+* `table_option` 目前支持的只有 `AUTO_INCREMENT`, `SHARD_ROW_ID_BITS`(详情可参考 [TiDB 专用系统变量和语法](/dev/reference/configuration/tidb-server/tidb-specific-variables.md#shard_row_id_bits) 中的介绍),
+ `PRE_SPLIT_REGIONS`，`CHARACTER SET` 和 `COMMENT`，其它只是语法上支持。具体内容参考下表，各个子句之间用逗号隔开。
 
     | 参数           |含义                                  |举例                        |
     |----------------|--------------------------------------|----------------------------|
     |`AUTO_INCREMENT`|自增字段初始值                        |`AUTO_INCREMENT` = 5|
+    |`SHARD_ROW_ID_BITS`|用来设置隐式 _tidb_rowid 的分片数量的 bit 位数 |`SHARD_ROW_ID_BITS` = 4|
+    |`PRE_SPLIT_REGIONS`|用来设置预先建立 Region 的个数 |`PRE_SPLIT_REGIONS` = 4|
```
Comment (crazycs520) [inline dev/reference/sql/statements/ddl.md]: 用来在建表时均匀切分 2^(PRE_SPLIT_REGIONS-1) 个 region。PRE_SPLIT_REGIONS 需要和搭配SHARD_ROW_ID_BITS 一起使用，且 PRE_SPLIT_REGIONS 必须小于 SHARD_ROW_ID_BITS。

**Comment by lilin90** [comment]
@zimulala Is this update for 3.0?

File: dev/reference/sql/statements/ddl.md [hunk 3]
```diff
@@ -170,14 +172,21 @@ table_option:
 
 * `index_option` 中 `KEY_BLOCK_SIZE` 目前只是语法上支持。
 
-* `table_option` 目前支持的只有 `AUTO_INCREMENT`，`CHARACTER SET` 和 `COMMENT`，其它只是语法上支持。具体内容参考下表，各个子句之间用逗号隔开。
+* `table_option` 目前支持的只有 `AUTO_INCREMENT`, `SHARD_ROW_ID_BITS`(详情可参考 [TiDB 专用系统变量和语法](/dev/reference/configuration/tidb-server/tidb-specific-variables.md#shard_row_id_bits) 中的介绍),
```
Comment (yikeke) [inline dev/reference/sql/statements/ddl.md]:
```suggestion
* `table_option` 目前支持的只有 `AUTO_INCREMENT`、`SHARD_ROW_ID_BITS`（详细介绍请参考 [TiDB 专用系统变量和语法](/dev/reference/configuration/tidb-server/tidb-specific-variables.md#shard_row_id_bits)）、
```
Please try not to use English commas in a Chinese document. If you want to list things, you can use "和" or enumeration commas (顿号).

File: dev/reference/sql/statements/ddl.md [hunk 4]
```diff
@@ -170,14 +172,21 @@ table_option:
...
+ `PRE_SPLIT_REGIONS`，`CHARACTER SET` 和 `COMMENT`，其它只是语法上支持。具体内容参考下表，各个子句之间用逗号隔开。
```
Comment (yikeke) [inline dev/reference/sql/statements/ddl.md]:
```suggestion
`PRE_SPLIT_REGIONS`、`CHARACTER SET` 和 `COMMENT`，其它只是语法上支持。具体内容参考下表，各个子句之间用逗号隔开。
```

**Comment by lilin90** [comment]
@zimulala Is this update for TiDB 3.0? OR just an ordinary update?

File: dev/reference/sql/statements/ddl.md (hunk 2 above)
Comment (zimulala) [reply dev/reference/sql/statements/ddl.md]: “PRE_SPLIT_REGIONS 需要和搭配SHARD_ROW_ID_BITS 一起使用” 目前实现不是这样的所以没有写，我觉得可以等配置里面介绍了 PRE_SPLIT_REGIONS 时，加个连接那里

**Comment by zimulala** [comment]
@lilin90 For TiDB 3.0
PTAL @yikeke @crazycs520 @bb7133

File: dev/reference/sql/statements/ddl.md (hunk 2 above)
Comment (crazycs520) [reply dev/reference/sql/statements/ddl.md]: ok

**Comment by bb7133** [comment]
> @zimulala Is this update for TiDB 3.0? OR just an ordinary update?

This is for 3.0

**Comment by bb7133** [comment]
LGTM, PTAL @lilin90 @yikeke

File: dev/reference/sql/statements/ddl.md [hunk 5]
```diff
@@ -185,15 +187,21 @@ table_option:
 
 * `index_option` 中 `KEY_BLOCK_SIZE` 目前只是语法上支持。
 
-* `table_option` 目前支持的只有 `AUTO_INCREMENT`、`CHARACTER SET`、`COLLATE` 和 `COMMENT`，其它只是语法上支持。具体内容参考下表，各个子句之间用逗号隔开。
+* `table_option` 目前支持的只有 `AUTO_INCREMENT`、`SHARD_ROW_ID_BITS`(详情介绍请参考 [TiDB 专用系统变量和语法](/dev/reference/configuration/tidb-server/tidb-specific-variables.md#shard_row_id_bits) 中的介绍),
```
Comment (yikeke) [inline dev/reference/sql/statements/ddl.md]:
```suggestion
* `table_option` 目前支持的只有 `AUTO_INCREMENT`、`SHARD_ROW_ID_BITS`（详情介绍请参考 [TiDB 专用系统变量和语法](/dev/reference/configuration/tidb-server/tidb-specific-variables.md#shard_row_id_bits)）、
```

**Comment by zimulala** [comment]
PTAL @yikeke

**Pull request merged**